APM > .NET > Logs injection: Remove reference to LibLog library (#9006)
DataDog/documentation: content/en/tracing/connect_logs_and_traces/dotnet.md

Enable injection in the .NET Tracer’s [configuration][1] by setting `DD_LOGS_INJECTION=true` through environment variables or the configuration files.

> The .NET Tracer uses the [LibLog][2] library to automatically inject trace IDs, span IDs, `env`, `service`, and `version` into your application logs. If you haven't done so already, it is recommended to configure the .NET tracer with `DD_ENV`, `DD_SERVICE`, and `DD_VERSION`. This will provide the best experience when adding `env`, `service`, and `version` (see [Unified Service Tagging][3] for more details).

Reviewer: Did you remove "into your application logs" at the end of the sentence intentionally? I don't think the sentence makes much sense without that.

Author: Good catch!

Author: Updated!

> The .NET Tracer can automatically inject trace IDs, span IDs, `env`, `service`, and `version` into your application logs. If you haven't done so already, it is recommended to configure the .NET tracer with `DD_ENV`, `DD_SERVICE`, and `DD_VERSION`. This will provide the best experience when adding `env`, `service`, and `version` (see [Unified Service Tagging][3] for more details).

Reviewer 2: ```suggestion
The .NET Tracer can automatically inject trace IDs, span IDs, `env`, `service`, and `version` into your application logs. If you haven't done so already, Datadog recommends configuring the .NET tracer with `DD_ENV`, `DD_SERVICE`, and `DD_VERSION`. This provides the best experience when adding `env`, `service`, and `version` (see [Unified Service Tagging][3] for more details).
```
